examples/framework-nextjs-app-directory/package.json

```diff
@@ -14,16 +14,16 @@
     "start": "next start"
   },
   "dependencies": {
-    "@keystone-6/auth": "^7.0.0",
-    "@keystone-6/core": "^5.7.0",
+    "@keystone-6/auth": "^7.0.2",
+    "@keystone-6/core": "^5.7.2",
     "@keystone-6/document-renderer": "^1.1.0",
-    "@keystone-6/fields-document": "^8.0.0",
+    "@keystone-6/fields-document": "workspace:^",
     "@preconstruct/next": "^4.0.0",
     "@prisma/client": "^4.16.2",
     "graphql": "^16.8.1",
     "graphql-request": "^5.0.0",
     "graphql-yoga": "^3.1.0",
-    "next": "^13.3.0",
+    "next": "^13.4.9",
```

Comment:
`next:build` builds fine with latest `nextjs` (14.1.0), however `keystone:build` has some unrelated problem with >=14
```
Error: <Html> should not be imported outside of pages/_document.
```

Reply:
Can we break this out into it's own pull request, and why do we need to bump?

Reply:
There is no real reason for this change. Originally I wanted to put `^14.1.0` here, but because I cannot we can just leave it as it was.